gradle/libs.versions.toml

```diff
@@ -118,6 +118,7 @@ datanucleus-jdo = "3.2.0-m3"
 hudi = "0.15.0"
 google-auth = "1.28.0"
 aliyun-credentials = "0.3.12"
+jcasbin = "1.81.0"
```

Comment:
Could you add this file's license and notice to License.bin and Notice.bin?

Reply:
It only needs to be listed in the license files, nothing needs to be added to notice.